src/common/transformations/include/transformations/common_optimizations/convert_u4_weights_zero_point_to_scalar.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertU4WeightsZeroPointToScalar;
```

Comment:
ConvertU4WeightsZeroPointToScalar -> ConvertWeightsZeroPointToScalar
and file name convert_u4_weights_zero_point_to_scalar.hpp -> convert_weights_zero_point_to_scalar.hpp